applications/GeoMechanicsApplication/tests/test_k0_procedure_process.py

```diff
@@ -8,6 +8,24 @@ class KratosGeoMechanicsK0ProcedureProcessTests(KratosUnittest.TestCase):
     """
     This class contains tests which check the result of a K0 Procedure Process
     """
+    def assert_stresses_at_integration_point(self, cauchy_stress_tensors, integration_point, expected_horizontal_stress, expected_vertical_stress, rel_tol):
+        """
+        Verifies whether the computed stresses are (nearly) equal to some expected values at the
+        given integration point.  Note that this function assumes there are no shear stresses!
```

Comment:
This might be a bit more clear (same with the other tests)
```suggestion
        given integration point.  Note that this test assumes there are no shear stresses!
```

Reply:
What I tried to express here is that function `assert_stresses_at_integration_point` is intended to check the stress state after applying the K0 procedure. In other words, it cannot be used to check _any_ stress state, but only the ones produced by the K0 procedure (which, by definition, will yield shear stresses that are equal to zero). From that point of view, it is not the test which assumes there are no shear stresses, but this check function. The reason for this comment is to avoid that this check function will be accidentally reused to verify stress states that were not produced by the K0 procedure. Therefore, I think we should not replace "function" by "test". We could, however, replace "function" by "checker" or "assertion". Would that be clearer in your opinion?

Reply:
Okay clear, in that case indeed the current comment is as it should be! I misread and thought this was a test as well, but indeed I now see this is a helper function